packages/ts/src/core/component/index.ts

```diff
@@ -151,4 +147,8 @@ export class ComponentCore<
   public isDestroyed (): boolean {
     return !this.element
   }
+
+  public getAriaDescription (): string {
```

Comment:
I think it's worth adding a comment about this method and why it return an empty string here in the base class

Reply:
The only requirement for this method to be present in the base class is basically to support inheritance, so that we can call the derived class method via polymorphism